marketing/views.py

```diff
@@ -143,48 +144,30 @@ def post(self, request):
         return Response({"message": "invalid data"}, status=status.HTTP_400_BAD_REQUEST)
 
     def get(self, request):
-        data_count = self.querysets.count()
-        params = request.query_params.dict()
-
-        if params.get("id"):
-            querysets = self.querysets.filter(id=params.get("id"))
-            jobs = JobGetSerializer(querysets, many=True).data
-            return Response(
-                {"data": jobs, "total_count": data_count}, status=status.HTTP_200_OK
-            )
-
-        if params.get("title"):
-            querysets = self.querysets.filter(title__icontains=params.get("title"))
-            jobs = JobGetSerializer(querysets, many=True).data
-            return Response(
-                {"data": jobs, "total_count": data_count}, status=status.HTTP_200_OK
-            )
-        
-        ############# Changed this for company ############### 
-        if params.get("company"):
-            querysets = self.querysets.filter(company=params.get("company"))
-            jobs = JobGetSerializer(querysets, many=True).data
-            return Response(
-                {"data": jobs, "total_count": data_count}, status=status.HTTP_200_OK
-            )
-
-        ############# Changed this for work location ############### 
-        if params.get("work_location"):
-            querysets = self.querysets.filter(work_location=params.get("work_location"))
-            jobs = JobGetSerializer(querysets, many=True).data
-            return Response(
-                {"data": jobs, "total_count": data_count}, status=status.HTTP_200_OK
-            )
+        querysets = self.querysets
+        params = request.query_params
```

Comment:
Why we have changed these get methods and it's params?
please i would recommend not to change them until unless I suggest so just add pagination